### PR TITLE
Some additions to the Day Length mod for BotW.

### PR DESCRIPTION
**Pull request by chrispurnell**

### CODE REVIEW

File: src/BreathOfTheWild/Mods/DayLength/rules.txt [hunk 1]
```diff
@@ -29,99 +31,169 @@ name = 15 seconds
...
+name = Scaled with day length
+category = Cloud Speed
+condition = ($timeCycleMode == 1)
+default = 1
+$cloudMultiplier = 0
+
+
+[Preset]
+name = 5 minutes
+category = Clock Interval
```
Comment (Crementif) [inline src/BreathOfTheWild/Mods/DayLength/rules.txt:131]: Just to be nitpicky, but I think this name could be a bit more descriptive perhaps. Ultimately up to you, but something like Clock Update Frequency, Clock Increment Rate or something else could work.

File: src/BreathOfTheWild/Mods/DayLength/rules.txt [hunk 2]
```diff
@@ -29,99 +31,169 @@ name = 15 seconds
...
+condition = ($timeCycleMode == 1)
+default = 1
+$clockAdjust:int = 1
+
+[Preset]
+name = 1 minute
+category = Clock Interval
```
Comment (Crementif) [inline src/BreathOfTheWild/Mods/DayLength/rules.txt:138]: Same as above

File: src/BreathOfTheWild/Mods/DayLength/rules.txt [hunk 3]
```diff
@@ -29,99 +31,169 @@ name = 15 seconds
...
+[Preset]
+name = Scaled with day length
+category = Cloud Speed
+condition = ($timeCycleMode == 1)
+default = 1
+$cloudMultiplier = 0
+
+
+[Preset]
+name = 5 minutes
```
Comment (Crementif) [inline src/BreathOfTheWild/Mods/DayLength/rules.txt:130]:
```suggestion
name = Increment Clock By 5 Minutes (Default)
```
I think this'd be a bit clearer for a user. The dropdown names can be pretty long.

File: src/BreathOfTheWild/Mods/DayLength/rules.txt [hunk 4]
```diff
@@ -29,99 +31,169 @@ name = 15 seconds
...
+category = Clock Interval
+condition = ($timeCycleMode == 1)
+default = 1
+$clockAdjust:int = 1
+
+[Preset]
+name = 1 minute
```
Comment (Crementif) [inline src/BreathOfTheWild/Mods/DayLength/rules.txt:137]:
```suggestion
name = Increment Clock By 1 Minute
```
Could go with every minute too, but I think it's easier to parse if you use the same formatting for both. Just my thoughts.

**Review by Crementif — COMMENTED** [review]
These are some cool improvements, nice work! I tried it in-game and it seems to work great.

I do have some minor nitpicks to make things a bit more descriptive, maybe. Let me know if you agree with them or feel free to do your own thing. After that I'll merge it.

File: src/BreathOfTheWild/Mods/DayLength/rules.txt (hunk 4 above)
Comment (chrispurnell) [reply src/BreathOfTheWild/Mods/DayLength/rules.txt:137]: I was unsure about what to put for a lot of that label text.  Your suggestions all seem good.
The other things I was unsure about was the exact values for the blood moon presets or if I should add other cloud speed presets.

Comment (Crementif) [reply src/BreathOfTheWild/Mods/DayLength/rules.txt:137]: I think with the amount of presets and types of presets I'd generally just go with what makes sense to you and you see someone using.

**Comment by Crementif** [comment]
Looks good to me, you want me to merge it?

**Comment by chrispurnell** [comment]
Yes please.

**Pull request merged**